dandiapi/api/views/users.py

```diff
@@ -78,5 +78,15 @@ def users_search_view(request: Request) -> HttpResponseBase:
     social_accounts = SocialAccount.objects.filter(extra_data__icontains=username)[:10]
     users = [social_account_to_dict(social_account) for social_account in social_accounts]
 
+    # Try searching Django's regular `User`s if there aren't any results.
+    # This allows this feature to work in development without having to conditionalize
+    # code based on the type of deployment.
+    if len(users) == 0:
```

Comment:
```suggestion
    if not users:
```
Apparently this is more pythonic https://stackoverflow.com/a/53522

Reply:
Changed it